### PR TITLE
Fixed issue #19713: Botstrap Buttons Multiple - When unchecking an exclusive option, the other options don't get re-enabled

### PR DESCRIPTION
**Pull request by gabrieljenik**

### CODE REVIEW

File: assets/packages/limesurvey/survey.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ function triggerEmRelevanceSubQuestion(){
                     $(item).prop("disabled", false );
                 });
             } else {
-                $(event.target).find('.answer-item input').each(function(itrt, item ){
+                $(event.target).find('.answer-item input, input.button-item').each(function(itrt, item ){
```
Comment (Shnoulle) [inline assets/packages/limesurvey/survey.js:69]: My opinion : twig file must be updated, it's an issue in theme.

Else : next time we add `input.my_item, .mybroken-theme-item .tag`

Can you try adding answer-item class at https://github.com/LimeSurvey/LimeSurvey/blob/59d73eeb8067ef5f916341274c670eda33aaa5fd/themes/question/bootstrap_buttons_multi/survey/questions/answer/multiplechoice/rows/answer_row.twig#L24 ?

**Review by Shnoulle — COMMENTED** [review]
Need to fix theme, to avoid adding more and more class.

**Comment by Shnoulle** [comment]
answer_item class **must** be at https://github.com/LimeSurvey/LimeSurvey/blob/59d73eeb8067ef5f916341274c670eda33aaa5fd/themes/question/bootstrap_buttons_multi/survey/questions/answer/multiplechoice/rows/answer_row.twig#L23 

Like the original one https://github.com/LimeSurvey/LimeSurvey/blob/59d73eeb8067ef5f916341274c670eda33aaa5fd/application/views/survey/questions/answer/multiplechoice/rows/answer_row.twig#L22

**Comment by gabrieljenik** [comment]
> answer_item class must be at

What about the question themes that were already created based on this? Wouldn't that be a backward incompratibility / or kind of like a break in the interface?

**Comment by Shnoulle** [comment]
> > answer_item class must be at
> 
> What about the question themes that were already created based on this? Wouldn't that be a backward incompratibility / or kind of like a break in the interface?

It's an issue in question theme, i don't understand why we need to update core for this.

Again : if question theme developper create any class name without reading https://manual.limesurvey.org/Theme_editor#Global_classes_for_question_part we MUST NOT fix it in core.

next time : `input.my_item, .mybroken-theme-item .tag` ?

**Comment by gabrieljenik** [comment]
I agree a change in theme is needed. I believe there's too much risk of doing it in master (lot of unexpected) on a minor release. I would solve it like this now and then in dev make it straight.

**Comment by tiborpacalat** [comment]
This has already been resolved with another ticket, sorry :(

**Comment by Shnoulle** [comment]
> This has already been resolved with another ticket, sorry :(

Can you show the fix ?

**Pull request closed without merging**